### DISCUSSION TITLE
MySQL CREATE UNIQUE INDEX

### DISCUSSION THREAD

**Discussion by beremour** [ClickHouse/ClickHouse#29029] [Q&A] [answered]
I am using native mysql client to connect to port 9004 and create application schema (zabbix monitoring software). I've got errors such as:
```
mysql> CREATE UNIQUE INDEX `role_1` ON `role` (`name`);
ERROR 62 (00000): Code: 62. DB::Exception: Syntax error: failed at position 8 ('UNIQUE'): UNIQUE INDEX `role_1` ON `role` (`name`). Expected one of: VIEW, LIVE, DATABASE, USER, PROFILE, TEMPORARY, TABLE, ROW POLICY, POLICY, DICTIONARY, QUOTA, OR REPLACE, ROLE, SETTINGS PROFILE, MATERIALIZED. (SYNTAX_ERROR) (version 21.9.2.17 (official build))
mysql>
```
How should I transform this statement ? Is it possible to find more mysql emulation info ?

**Comment by den-crane** [comment, marked as answer]
It won't work. Clickhouse is not OLTP database.
Use https://glaber.io/downloads/

**Reply by beremour** [reply]
Thank you! I am following this project, but it's too buggy for now. Last version binaries just crashes on my system. I can't use it in production.
